index.html

```diff
@@ -320,15 +320,18 @@ <h5 class="ui header">Why Did Apple and Nvidia Stock Open Lower on Thursday?</h5
         <div class="six wide right aligned column shares-container">
           <div class="ui fluid input">
             <input id="sell-shares" class="right" type="text" placeholder="0" maxlength="4">
+              / <span id="total-shares">
           </div>
         </div>
         <div class="ten wide column">Market Price</div>
         <div class="six wide right aligned column">$<span id="sell-price"></span></div>
       </div>
       <div class="ui divider shares-divider"></div>
       <div class="ui grid">
-        <div class="ten wide column">Total</div>
+        <div class="ten wide column">Selling</div>
```

Comment:
For some reason I don't really like selling as a name here, but idk what else to change it to